### PR TITLE
Add questions and demographic questions

### PR DESCRIPTION
**Pull request by drewpotter**
Initial work towards adding questions and demographic questions. Opening a PR so that @spencerldixon can add comments. I still need to fix the tests and also I need to protect the questions and demographic questions from non admin access.

### CODE REVIEW

File: app/controllers/surveys_controller.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def destroy
   private
     # Use callbacks to share common setup or constraints between actions.
     def set_survey
-      @survey = Survey.find_by_uuid(params[:uuid])
+      @survey = Survey.find_by(uuid: params[:uuid])
```
Comment (spencerldixon) [inline app/controllers/surveys_controller.rb]: Why

File: app/helpers/demographic_questions_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module DemographicQuestionsHelper
```
Comment (spencerldixon) [inline app/helpers/demographic_questions_helper.rb]: Might as well remove these

File: app/controllers/surveys_controller.rb (hunk 1 above)
Comment (drewpotter) [reply app/controllers/surveys_controller.rb]: Uhm I have no idea why this got changed back @spencerldixon unless I needed to merge with master before I pushing this? I will revert this change immediately.

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -17,4 +18,8 @@ def configure_permitted_parameters
     devise_parameter_sanitizer.permit(:sign_up, keys: keys)
     devise_parameter_sanitizer.permit(:account_update, keys: keys)
   end
+
+  def require_admin
+    redirect_to root_path unless current_user.admin?
```
Comment (spencerldixon) [inline app/controllers/application_controller.rb]: Let's add a notice in here to say why

Comment (drewpotter) [reply app/controllers/application_controller.rb]: Thanks @spencerldixon good point!

**Pull request merged**